src/shared/utilities/vsCodeUtils.ts

```diff
@@ -6,100 +6,118 @@
 'use strict'
 import * as vscode from 'vscode'
 import * as nls from 'vscode-nls'
-export const localize = nls.loadMessageBundle()
 
-import { ErrorOrString, getLogger, Logger, LogLevel } from '../logger'
+export const localize: TemplateParser = nls.loadMessageBundle()
 
-// ------- Experimental combined output channel & traditional logger  --------------
+// TODO: initialize NLS here rather than extension.ts. Prefer packages to import localize from here
```

Comment:
This is opinionated and lacks a proper design.
Remove these comments.
Create a design task in the backlog that identifies the requirements you know at this time.
A improvement can be discussed and planned out to provide a more directed change.

Reply:
Adjusted comment and create associated task: https://github.com/aws/aws-toolkit-vscode/issues/353